setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup, find_namespace_packages
+
+setup(
+    name="lsst.ts.salobj",
+    description="Object-oriented interface to LSST SAL communications middleware.",
+    use_scm_version=True,
+    setup_requires=["setuptools-scm", "pytest-runner"],
+    install_requires=[],
+    package_dir={"": "python"},
+    packages=find_namespace_packages(where="python"),
+    package_data={"": ["*.rst", "*.yaml"]},
+    scripts=["bin.src/purge_topics.py", "bin.src/run_test_csc.py", "bin.src/show_salpy_attributes.py"],
+    test_requires=["pytest", "pytest-cov", "pytest-flake8"],
+    license="GPL",
+    project_urls={
+        "Bug Tracker": "https://jira.lsstcorp.org/secure/Dashboard.jspa",
+        "Source Code": "https://github.com/lsst-ts/ts_salobj",
+    }
+)
```

Comment:
```
"""Sphinx configuration file for an LSST stack package.

This configuration only affects single-package Sphinx documentation builds.
"""

from documenteer.sphinxconfig.stackconf import build_package_configs
import lsst.ts.salobj


_g = globals()
_g.update(build_package_configs(
    project_name='ts_salobj',
    version=lsst.ts.salobj.version))
```

I also changed the version to reflect the version.py changes.

Reply:
Shouldn't this be `version=lsst.ts.salobj.version.__version__`?